File: backend/CaseTecnico.MRA.Application/UseCases/ArquivoNaoRecepcionados/GetArquivoNaoRecepcionadoDatatable/GetArquivoNaoRecepcionadoDatatableResponse.cs

namespace CaseTecnico.MRA.Application.UseCases.ArquivoNaoRecepcionados.GetArquivoNaoRecepcionadoDatatable;

public class GetArquivoNaoRecepcionadoDatatableResponse
{
    public List<GetArquivoNaoRecepcionadoDatatableDto> Data { get; set; } = new();

    // ----- Paginação -----
    public int TotalRecords { get; set; }
    public int Page { get; set; }
    public int PageSize { get; set; }
}

File: frontend/src/app/modules/arquivo/models/responses/get-arquivo-nao-recepcionado-datatable-response.ts
import { GetArquivoNaoRecepcionadoDatatableDto } from "../dtos/get-arquivo-nao-recepcionado-datatable.dto";

export class GetArquivoNaoRecepcionadoDatatableResponse {
    data?: GetArquivoNaoRecepcionadoDatatableDto[];
    totalRecords?: number;
    page?:number;
    pageSize?:number;
}